pynamodb/_compat.py

```diff
@@ -2,7 +2,16 @@
 
 if six.PY2:
     from inspect import getargspec as getfullargspec
+    from imp import load_source
 else:
     from inspect import getfullargspec
+    from importlib.machinery import SourceFileLoader
+
+def load_source_compat(name, path):
+    """Load module using the Python version compatible function. """
```

Comment:
```suggestion
    """Load module using the Python version compatible function."""
```

Also, maybe name it `load_module`, similar to how preexisting `getfullargspec` isn't exported as `getfullargspec_compat`?